tests/aggregation_regress/models.py

```diff
@@ -104,3 +104,8 @@ class Bravo(models.Model):
 class Charlie(models.Model):
     alfa = models.ForeignKey(Alfa, null=True)
     bravo = models.ForeignKey(Bravo, null=True)
+
+
+class Thing(models.Model):
```

Comment:
IMO, would be helpful to use a more descriptive model name like SelfReferentialFK - what do you think?